### PR TITLE
[book] controller ch review, part 1

### PR DESCRIPTION
**Pull request by talitakz**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | all |
| Fixed tickets | x |
- tried to better form the concept of controller methods and controller classes - nothing new was edded just better formed
- changes in styling, wording, typos

### CODE REVIEW

**Comment by weaverryan** [comment]
WOW! I'll review all of your pr's this week!

**Comment by paxyknox** [comment]
Thank you! I have also Doctrine in Template chapter in store but first this.
I hope the PR are small enough! :)

File: book/controller.rst [hunk 1]
```diff
@@ -143,7 +150,7 @@ Mapping a URL to a Controller
 
 The new controller returns a simple HTML page. To actually view this page
 in your browser, you need to create a route, which maps a specific URL path
-to the controller:
+to the controller::
```
Comment (weaverryan) [inline book/controller.rst]: It's only 2 `::` right before a `php` block (and then we don't need the `.. code-block:: php`

Comment (paxyknox) [inline book/controller.rst]: Noted!

File: book/controller.rst [hunk 2]
```diff
@@ -211,13 +218,13 @@ simply creating a controller method and an associated route.
 
 Simple, right?
 
-.. sidebar:: The AppBundle:Hello:index controller syntax
-
-    If you use the YML or XML formats, you'll refer to the controller using
-    a special shortcut syntax: ``AppBundle:Hello:index``. For more details
-    on the controller format, see :ref:`controller-string-syntax`.
+.. sidebar:: Logical controller name
```
Comment (weaverryan) [inline book/controller.rst]: I'm reversing this - I think the user will recognize what we mean by `AppBundle:Hello:index` - but they may not know what a "logical name" is at this point :)

File: book/controller.rst [hunk 3]
```diff
@@ -307,62 +318,58 @@ Take the following more-interesting example:
...
 
-.. tip::
+    You can also pass other variables from your route to your controller
+    arguments. See :doc:`/cookbook/routing/extra_information`.
 
-    Every route also has a special ``_route`` parameter, which is equal to
-    the name of the route that was matched (e.g. ``hello``). Though not usually
-    useful, this is also available as a controller argument. You can also
-    pass other variables from your route to your controller arguments. See
-    :doc:`/cookbook/routing/extra_information`.
```
Comment (weaverryan) [inline book/controller.rst]: I like what you did with this entire previous section :)

**Comment by weaverryan** [comment]
Hey @paxyknox!

Phew! Ok I just merged everything in and made some changes here: sha: 6214eb668c8dd4f1c18c30acabe456fbe65ebe3c. If you see anything weird, let me know or (better) make a new PR. I like most of your changes - and just made additional changes to some of them.

And yes, the PR's were pretty easy to review :). Here's what I can tell you:

1) Just make 1 PR for 1 chapter (instead of multiple). Because these cover so much, but don't make HUGE changes, I merge them and then review the sections you changed. I merged all 3 PR's at once, and they conflicted just a little with each other. So, just make one :).

2) Re-wording is always subjective. If you are sure that something should be re-worded, then do it! But if you're not sure if your new wording is a big improvement, skip it: that saves us time reviewing and merging some "sideways" changes (things that are maybe just "slightly" better... or maybe not better at all).

Thanks!

**Comment by paxyknox** [comment]
I reviewed the merge it looks perfect! :)

Thank you for this instructions on how to make better PR, I really appreciate it!

**Pull request merged**